package.json

```diff
@@ -62,7 +62,7 @@
     "whatwg-fetch": "1.0.0"
   },
   "scripts": {
-    "preinstall": "if ! [ -d elastic-apm-rum ]; then git clone --branch='@elastic/apm-rum@4.8.1' https://github.com/elastic/apm-agent-rum-js elastic-apm-rum --depth 1; fi",
+    "preinstall": "if ! [ -d elastic-apm-rum ]; then (git clone https://github.com/elastic/apm-agent-rum-js elastic-apm-rum --depth 1; cd elastic-apm-rum; git fetch origin '@elastic/apm-rum@4.8.1'; git reset --hard FETCH_HEAD) fi",
```

Comment:
git fetch origin '@elastic/apm-rum@4.8.1' , will this work?

Reply:
Asking because is it set to remote or origin?

Reply:
See the below output when using a branch/tag and using a sha commit

```
$ git clone https://github.com/elastic/apm-agent-rum-js elastic-apm-rum --depth 1 ; \
 cd elastic-apm-rum/ ; \
 git fetch origin @elastic/apm-rum@4.8.1 ; \
 git reset --hard FETCH_HEAD ; \
 git log -1 ; \
 git remote -v

Cloning into 'elastic-apm-rum'...
remote: Enumerating objects: 350, done.
remote: Counting objects: 100% (350/350), done.
remote: Compressing objects: 100% (318/318), done.
remote: Total 350 (delta 120), reused 99 (delta 21), pack-reused 0
Receiving objects: 100% (350/350), 486.41 KiB | 1.32 MiB/s, done.
Resolving deltas: 100% (120/120), done.
remote: Enumerating objects: 4617, done.
remote: Counting objects: 100% (4617/4617), done.
remote: Compressing objects: 100% (1320/1320), done.
remote: Total 4384 (delta 3222), reused 4191 (delta 3030), pack-reused 0
Receiving objects: 100% (4384/4384), 2.76 MiB | 4.13 MiB/s, done.
Resolving deltas: 100% (3222/3222), completed with 149 local objects.
From https://github.com/elastic/apm-agent-rum-js
 * tag               @elastic/apm-rum@4.8.1 -> FETCH_HEAD
HEAD is now at 2ef506a chore(release): publish
commit 2ef506ab1fc3a92d643e96edf0e226ac39b238e4 (HEAD -> master)
Author: Hamid <h.jahtalab@gmail.com>
Date:   Fri Feb 14 15:54:14 2020 +0100

    chore(release): publish
    
     - @elastic/apm-rum-angular@0.4.3
     - @elastic/apm-rum-core@4.9.1
     - @elastic/apm-rum-react@0.3.6
     - @elastic/apm-rum-vue@0.3.2
     - @elastic/apm-rum@4.8.1
origin	https://github.com/elastic/apm-agent-rum-js (fetch)
origin	git@github.com:elastic/apm-agent-rum-js (push)

```

```
$ git clone https://github.com/elastic/apm-agent-rum-js elastic-apm-rum --depth 1 ; \
 cd elastic-apm-rum/ ; \
 git fetch origin 03b4781d10fd9a8d1b71a0cf1ed685ae0a3296ab ; \
 git reset --hard FETCH_HEAD ; \
 git log -1 ; \
 git remote -v

Cloning into 'elastic-apm-rum'...
remote: Enumerating objects: 350, done.
remote: Counting objects: 100% (350/350), done.
remote: Compressing objects: 100% (318/318), done.
remote: Total 350 (delta 120), reused 99 (delta 21), pack-reused 0
Receiving objects: 100% (350/350), 486.41 KiB | 245.00 KiB/s, done.
Resolving deltas: 100% (120/120), done.
remote: Enumerating objects: 4620, done.
remote: Counting objects: 100% (4620/4620), done.
remote: Compressing objects: 100% (1320/1320), done.
remote: Total 4387 (delta 3225), reused 4195 (delta 3033), pack-reused 0
Receiving objects: 100% (4387/4387), 2.76 MiB | 4.55 MiB/s, done.
Resolving deltas: 100% (3225/3225), completed with 149 local objects.
From https://github.com/elastic/apm-agent-rum-js
 * branch            03b4781d10fd9a8d1b71a0cf1ed685ae0a3296ab -> FETCH_HEAD
HEAD is now at 03b4781 docs: clean up distributed tracing
commit 03b4781d10fd9a8d1b71a0cf1ed685ae0a3296ab (HEAD -> master)
Author: Brandon Morelli <brandon.morelli@elastic.co>
Date:   Tue Feb 18 16:07:17 2020 -0800

    docs: clean up distributed tracing
origin	https://github.com/elastic/apm-agent-rum-js (fetch)
origin	git@github.com:elastic/apm-agent-rum-js (push)
```


There is one origin only, what do you mean with remote?

Reply:
Cool, I was since it could be coming from forks. But yeah we only have origin here. Thanks for the logs.